engine/installation/linux/oracle.md

```diff
@@ -94,7 +116,27 @@ Docker from the repository.
     ```bash
     $ sudo yum -y check-update
     ```
-2.  Install the latest version of Docker, or go to the next step to install a
+
+2.  Verify and import Docker's public key, which is used to sign packages in
```

Comment:
This entire step is not required because you specify the GPG key in the YUM repository configuration and you've enabled GPG checking as part of the yum install process. Therefore, the install of the GPG will automatically be triggered by YUM when the package is installed. 

Thus, this step should be removed and an additional step should be added later to verify the GPG key when the `yum` command prompts the user to accept it.